core-ui/src/components/Predefined/Create/AddonsConfigurations/AddonsConfigurations.create.js

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
 import { AddonsConfigurations } from './AddonsConfigurations.js';
+import { useTranslation } from 'react-i18next';
 
-export const AddonsConfigurationsCreate = ({ resourceType, ...props }) => (
-  <AddonsConfigurations resourceType={resourceType} {...props} />
-);
+export const AddonsConfigurationsCreate = ({ resourceType, ...props }) => {
+  const { t } = useTranslation();
+
+  return (
+    <AddonsConfigurations
+      resourceType={resourceType}
+      kind={t('addons.configuration')}
```

Comment:
The whole`Addon Configuration` is a resource name. So I'd change the yaml structure to be consistent to`'addons.singular_name'` for example instead of `'addons.configuration'`.